Web/Views/Usuario/_Cadastrar.cshtml

```diff
@@ -0,0 +1,41 @@
+@{
+    Layout = null;
+}
+
+<div class="cadastrar-container">
+  <form id="cadastroUsuario">
+      <h1>Crie sua conta</h1>
+      <hr>
+
+      <label for="nome"><b>Nome</b></label>
```

Comment:
Para todo o formulario, podemos adotar o padrão asp-for para as lable, assim conseguimos aproveitar algumas funções do aspnet junto com o javascript.
O padrão Krita usa-se do jeito que vc fez, mas eu acredito ser uma melhoria do jeito que estou propondo.